### PR TITLE
Ams style check and refactor

### PR DESCRIPTION
**Pull request by mateusz-ciesielski**

### CODE REVIEW

File: extras/ams_wrapper/src/api/models/model_builder.py [hunk 1]
```diff
@@ -31,10 +33,12 @@ def load_model_config(cls, config_file_path: str) -> dict:
                 config = json.load(config_file)
         except FileNotFoundError as e:
             # TODO: think what exactly should we do in this case
```
Comment (rasapala) [inline extras/ams_wrapper/src/api/models/model_builder.py:35]: Please remove.

File: extras/ams_wrapper/src/api/types.py [hunk 2]
```diff
@@ -126,14 +137,17 @@ def as_dict(self):
...
 
+
 class InferenceExtension:
     # TODO: add support
     pass
 
+
 class Inference:
     def __init__(self, value: ResultType, extensions: List[InferenceExtension] = None):
         self.value = value
```
Comment (rasapala) [inline extras/ams_wrapper/src/api/types.py:153]: Plz remove.

File: extras/ams_wrapper/tests/accuracy/vehicle_detection_accuracy.py [hunk 3]
```diff
@@ -1,97 +1,124 @@
...
 from tensorflow_serving.apis import prediction_service_pb2_grpc
 import grpc
-import json
 import requests
 
 from extras.ams_wrapper.src.preprocessing.preprocess_image import preprocess_binary_image
 
 test_image = "tests/functional/fixtures/test_images/1car_fit.jpeg"
-#test_image = "tests/functional/fixtures/test_images/street.jpeg"
+# test_image = "tests/functional/fixtures/test_images/street.jpeg"
```
Comment (rasapala) [inline extras/ams_wrapper/tests/accuracy/vehicle_detection_accuracy.py:30]: Please remove commented code.

File: extras/ams_wrapper/tests/accuracy/vehicle_detection_accuracy.py [hunk 4]
```diff
@@ -1,97 +1,124 @@
...
+            image_bytes, reverse_input_channels=True)
     return image
 
+
 def send_image_ovms(file_path):
-    channel = grpc.insecure_channel("{}:{}".format("localhost","9000"))
+    channel = grpc.insecure_channel("{}:{}".format("localhost", "9000"))
     stub = prediction_service_pb2_grpc.PredictionServiceStub(channel)
-    #img = load_tf_image(file_path).reshape(1,3,384, 672)
+    # img = load_tf_image(file_path).reshape(1,3,384, 672)
```
Comment (rasapala) [inline extras/ams_wrapper/tests/accuracy/vehicle_detection_accuracy.py:54]: Commented code.

**Comment by mateusz-ciesielski** [comment]
@rasapala commented out code was removed

**Pull request merged**